LawGuardPro.API/LawGuardPro.API.csproj

```diff
@@ -15,6 +15,7 @@
   <ItemGroup>
     <PackageReference Include="MediatR" Version="12.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.5" />
+    <PackageReference Include="Microsoft.AspNetCore.JsonPatch" Version="8.0.5" />
```

Comment:
Remove this if not needed

Reply:
Removed unnecessary package reference from csproj